scripts/rename-schema.sh

```diff
@@ -7,6 +7,8 @@ perl -pi -e s,'FacetedCompositeUnitSpec','FacetedUnitSpec',g build/vega-lite-sch
 perl -pi -e s,'ExtendedLayerSpec','LayerSpec',g build/vega-lite-schema.json
 perl -pi -e s,'GenericLayerSpec<CompositeUnitSpec>','LayerSpec',g build/vega-lite-schema.json
 perl -pi -e s,'Generic(.*)<CompositeUnitSpec\,LayerSpec>','\1',g build/vega-lite-schema.json
+perl -pi -e s,'Generic(.*)<FacetedUnitSpec\,LayerSpec>','Faceted\1',g build/vega-lite-schema.json
```

Comment:
I don't think this line should be added.  VConcatSpec should remain its name, for example.

Reply:
Done